pkgs/top-level/aliases.nix

```diff
@@ -874,6 +874,8 @@ mapAliases ({
   linuxPackages_testing_bcachefs = throw "'linuxPackages_testing_bcachefs' has been removed, please use 'linuxPackages_latest', any kernel version at least 6.7, or any other linux kernel with bcachefs support";
   linux_testing_bcachefs = throw "'linux_testing_bcachefs' has been removed, please use 'linux_latest', any kernel version at least 6.7, or any other linux kernel with bcachefs support";
 
+  llvmPackages_git = (recurseIntoAttrs (callPackages ../development/compilers/llvm { })).git;
```

Comment:
This means `llvmPackages_git` is now deprecated and will be removed after a certain period of time. Are you sure about that?

Also Please add `Added YYYY-MM-DD` to support automation.

Reply:
No, we will not be deprecating `llvmPackages_git`. Alyssa was just wanting it moved back into `aliases.nix` since the LLVM commonification moved it out of there.